config/initializers/sneakers.rb

```diff
@@ -0,0 +1,8 @@
+require 'sneakers'
+require_relative 'messaging_config'
+Sneakers.configure(
+  amqp: Messaging.config['events']['server'],
+  exchange: Messaging.config['events']['exchange']['geoblacklight'],
+  exchange_type: :fanout
+)
+Sneakers.logger.level = Logger::INFO
```

Comment:
can we change this to `Logger::DEBUG` or something that shows us more information?